main.go

```diff
@@ -30,6 +30,7 @@ import (
 	"github.com/dell/csi-baremetal-operator/pkg"
 	"github.com/dell/csi-baremetal-operator/pkg/acrvalidator"
 	"github.com/dell/csi-baremetal-operator/pkg/common"
+	"github.com/sirupsen/logrus"
```

Comment:
You should place extra libraries imports in second section. [Reference](https://github.com/dell/csi-baremetal/blob/master/docs/CONTRIBUTING.md)